extensions/react-native.js

```diff
@@ -19,13 +19,14 @@ module.exports = context => {
     rules: {
       androidVersion: {
         check: async (rule, context) => {
+          const androidEnvData = await getAndroidEnvData(context)
```

Comment:
how come here you use an intermediate variable, and below in report you destructure directly from the awaited call?